src/annotator/test/integration/multi-frame-test.js

```diff
@@ -1,19 +1,18 @@
 import { DEBOUNCE_WAIT } from '../../frame-observer';
```

Comment:
This module could use a better name, eg. `integration/hypothesis-injector-test.js`.

Reply:
I agree, I replaced the name of the file.